chapters/introduction.tex

```diff
@@ -72,8 +72,33 @@ \section{Some Definitions}\label{some-definitions}
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Initialization]\index{initialization}
```

Comment:
I suppose the term being defined actually includes _problem_?
```suggestion
\begin{definition}[Initialization problem]\index{def:initialization-problem}
```

Reply:
I prefer to define the current shorter 'Initialization'; to me the logic is that the Simulation starts with the Initialization which means _solving_ the Initialization problem. I'm not sure if that needs to be made clearer.

When we later use it we use terms like 'during initialization'.